### PR TITLE
DOCSP-40414: jvm crypto improvement

### PR DESCRIPTION
**Pull request by rustagir**
[DOCSP-40414](https://jira.mongodb.org/browse/DOCSP-40414)

### CODE REVIEW

File: dbx/jvm/v5.2-wn-items.rst [hunk 1]
```diff
@@ -9,6 +9,32 @@
   can use a configured FIPS-compliant JCA provider to provide a higher
   level of security.
 
+- Performance improvements due to implementation of native cryptography
+  on all supported platforms. The following list describes the actions
+  needed to implement this improvement depending on your operating
+  system:
+  
+  - **Windows**: Upgrade to your ``mongodb-crypt`` version to v5.2.0.
```
Comment (mongoKart) [inline dbx/jvm/v5.2-wn-items.rst:17]: I: assuming this should be like this?

```suggestion
  - **Windows**: Upgrade your ``mongodb-crypt`` version to v5.2.0.
```

File: dbx/jvm/v5.2-wn-items.rst [hunk 2]
```diff
@@ -9,6 +9,32 @@
...
+
+  - **Mac**: Upgrade to your ``mongodb-crypt`` version to v5.2.0.
```
Comment (mongoKart) [inline dbx/jvm/v5.2-wn-items.rst:19]:
```suggestion
  - **Mac**: Upgrade your ``mongodb-crypt`` version to v5.2.0.
```

File: dbx/jvm/v5.2-wn-items.rst [hunk 3]
```diff
@@ -9,6 +9,32 @@
...
+
+  - **Linux**: Install ``libmongocrypt.so`` directly on the file system,
+    instead of using the file that is bundled within the
+    ``mongodb-crypt`` JAR file. View Linux instructions for installing
```
Comment (mongoKart) [inline dbx/jvm/v5.2-wn-items.rst:23]: S: maybe a little less direct

```suggestion
    ``mongodb-crypt`` JAR file. You can find Linux instructions for installing
```

File: dbx/jvm/v5.2-wn-items.rst [hunk 4]
```diff
@@ -9,6 +9,32 @@
...
+    ``libmongocrypt`` in the :manual:`Server manual
+    </core/csfle/reference/libmongocrypt/#linux-installation>`. If you
+    use a package manager to install ``libmongocrypt``, JNA will find it
```
Comment (mongoKart) [inline dbx/jvm/v5.2-wn-items.rst:26]: Q: Has JNA already been defined on this page/these pages?

**Review by mongoKart — CHANGES_REQUESTED** [review]
a few small things!

File: dbx/jvm/v5.2-wn-items.rst (hunk 4 above)
Comment (rustagir) [reply dbx/jvm/v5.2-wn-items.rst:26]: wrote it out

File: dbx/jvm/v5.2-wn-items.rst [hunk 5]
```diff
@@ -9,6 +9,32 @@
...
+    there without further configuration. If there is an issue, you can
+    specify the search path by setting ``LD_LIBRARY_PATH``.
```
[the remark this answers is not in the document]
Comment (rustagir) [reply dbx/jvm/v5.2-wn-items.rst:28]: I will tag tech reviewer to answer this

**Review by mongoKart — APPROVED** [review]
lgtm!

File: dbx/jvm/v5.2-wn-items.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (rustagir) [reply dbx/jvm/v5.2-wn-items.rst]: @jyemin

[the remark this answers is not in the document]
Comment (jyemin) [reply dbx/jvm/v5.2-wn-items.rst]: I would just change "If there is an issue" to: "Alternatively"

**Review by jyemin — APPROVED** [review]

**Pull request merged**